apps/shared/fonts.css

```diff
@@ -0,0 +1,45 @@
+@font-face {
+  font-family: 'IBM Plex Sans';
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: local('IBM Plex Sans Light'), local('IBMPlexSans-Light'),
+    url('./fonts/ibm-plex-sans-v7-latin-300.woff2') format('woff2'),
+    url('./fonts/ibm-plex-sans-v7-latin-300.woff') format('woff');
```

Comment:
Are these references processed by webpack? I.e. does it bundle the fonts on build?

Reply:
To be honest, I don't really know for sure. First guess is _no_ because neither the Webpack config in the Judicial system project f.x. nor the nrwlConfig webpack config handles font files. However, the Judicial system builds without a problem and generates font files in `/dist/apps/judicial-system` and `nx build` is running webpack under the hood. I'm guessing there is some magic happening here and that they actually are, even though I can't find where it's being done 🤷‍♂️